metric_system/functions/aws_metric_publisher.py

```diff
@@ -9,18 +8,15 @@
 class AwsMetricPublisher(MetricPublisher):
     """Publishes Metric-specific data"""
 
-    def __init__(self, name_space: str, instance_id: str, aws_region: str):
+    def __init__(self, name_space: str):
         """Instantiate the AWSMetricPublisher object.
 
         Args:
             name_space (str): The namespace of the metric.
-            instance_id (str): The ID of the instance associated with the metric.
             aws_region (str): The AWS region to use for CloudWatch.
```

Comment:
This line should be removed because you removed the parameter from the method header